server/src/main/java/org/opensearch/action/admin/cluster/stats/ClusterStatsResponse.java

```diff
@@ -88,12 +90,42 @@ public ClusterStatsResponse(
         List<ClusterStatsNodeResponse> nodes,
         List<FailedNodeException> failures,
         ClusterState state
+    ) {
+        this(
+            timestamp,
+            clusterUUID,
+            clusterName,
+            nodes,
+            failures,
+            state,
+            ClusterStatsRequest.Metric.allMetrics(),
+            ClusterStatsRequest.IndexMetrics.allIndicesMetrics()
```

Comment:
Is this required ?

Reply:
Yes, this change was done to avoid duplication of code.